include/kframework/evm.md

```diff
@@ -613,12 +608,14 @@ After executing a transaction, it's necessary to have the effect of the substate
            ...
          </message>
 
+    rule <k> (.K => #newAccount MINER) ~> #finalizeTx(_)... </k>
+         <coinbase> MINER </coinbase> [owise]
```

Comment:
I think we can just get rid of this rule? Because above the rule for `#finalizeTx(true) => ...` already injects `MINER` into the set  of accounts to call `#finalizeStorage` on, and `#finalizeStorage` will already make sure the account exists when it does its work.

Reply:
@ehildenb `#finalizeStorage` is adding the account, but after `#finalizeTx(true)`. The execution gets stuck because in the rule for `#finalizeTx(false => true)`, the coinbase account is not added in the state.